File: my js/pattern-problems/pattern-7.js
// const prompt = require("prompt-sync")();

let n = parseInt(prompt("Give me a number: "));

// let count = 1;
for (let row = 1; row <= n; row++) {
  for (let col = 1; col <= row; col++) {
    document.write(row);
  }
  document.write("<br/>");
}

// expected output
// if n = 7
1;
22;
333;
4444;
55555;
666666;
7777777;

File: my c++/pattern-problems/pattern-7.cpp
#include <iostream>
using namespace std;

int main(){
    int n;
    cin >> n;
    int row = 1;
    while (row<=n)
    {
        int col = 1;
        while (col<=row)
        {
            cout << row;
            col++;    
        }
        cout << endl;
        row++;
    }
    
}

// expected output
// if n = 4
// 1
// 22
// 333
// 4444